src/css/components/activity_log.scss

```diff
@@ -48,50 +44,35 @@
 .activity_log-item-error,
 .activity_log-item-success,
 .activity_log-item-warning {
-
-  padding-left: $grid-2;
-
-  &::before {
-    @extend .status-pill !optional;
-    height: 75%;
-  }
-
-  .activity_log-item_text {
-    font-family: $font-sans;
-    font-weight: 600;
+  .activity_log-item_line {
+    .activity_log-item_text {
+      font-family: $font-sans;
```

Comment:
Why is so much nesting needed? If it's hard to override, can we fix what's so high in specificity?